jackmatchmaker/jacklib.py

```diff
@@ -1243,9 +1244,27 @@ def get_time():
 # -------------------------------------------------------------------------------------------------
 # Misc
 
+global _error_callback
+_error_callback = None
+
 jlib.jack_free.argtypes = [c_void_p]
 jlib.jack_free.restype = None
 
+try:
+    jlib.jack_set_error_function.argtypes = [JackErrorCallback]
+    jlib.jack_set_error_function.restype = None
+except AttributeError:
+    jlib.jack_set_error_function = None
+
+
+def set_error_callback(error_callback):
```

Comment:
Please keep to the jack lib naming, i.e. `set_error_function` (even though it is inconsistent with the other callback setting functions).

We can provide an `set_error_callback` alias.

Reply:
Done, although I did not create an alias so feel free to add one if you think it's needed.